src/vsphere_cpi/spec/integration/vm_host_rule_spec.rb

```diff
@@ -198,8 +226,8 @@
               datastore_pattern: @datastore_shared_pattern,
               persistent_datastore_pattern: @datastore_shared_pattern,
               clusters: [
-                { @cluster_name => { host_group: @first_host_group } },
-                { @second_cluster_name => { host_group: @third_host_group } },
+                { @cluster_name => { host_group: {'name' => @first_host_group, 'drs_rule' => 'MUST'} } },
+                { @second_cluster_name => { host_group: {'name' => @third_host_group, 'drs_rule' => 'must'} } },
```

Comment:
Is this supposed to be lowercase?

Reply:
The code handles these `drs_rule` strings in a case free manner. 

That's why tests mention both to ensure robustness.